lua/Events.lua

```diff
@@ -1,10 +1,14 @@
+local triggerables = {}

```

Comment:
My biggest concern here is leaking the context of `triggerables` into the rest of the code, which could allow the variable to be overwritten, as AFAIK this file is executed globally. I might be wrong though, it's been a while, I don't recall 100% how DU events work.

Reply:
Another thing that slightly concerns me is how GC would work here. Let's say we create a bunch of objects with events and stop using them. Usually the Lua GC would eventually remove them from memory, but in this case, since we're referencing them somewhere, they might stay in memory and lead to a leak...

Reply:
Sounds okay, let's add it and see how it goes :)